### PR TITLE
Rouny takeover

### PR DESCRIPTION
**Pull request by Zar000**
## Changelog

:cl:
imageadd: added roooouny
admin: ya can rounify all xenos with 1 simple click in Fun tab
/:cl:

### CODE REVIEW

**Comment by ZahaSanko** [comment]
based PR speedmerge

**Comment by novaepee** [comment]
image
Ehh? Why are you deleting Warlock?

**Comment by Zar000** [comment]
> image
> Ehh? Why are you deleting Warlock?

Oh shit thanks, thought merge conflict resolved properly

**Comment by ClosetedSkeletonGH** [comment]
> > image
> > Ehh? Why are you deleting Warlock?
> 
> Oh shit thanks, thought merge conflict resolved properly

Warlocks walking animation is still broken

File: code/modules/admin/fun_verbs.dm [hunk 1]
```diff
@@ -68,6 +68,19 @@
 	log_admin("[key_name(usr)] created a Queen Mother report: [input]")
 	message_admins("[ADMIN_TPMONTY(usr)] created a Queen Mother report.")
 
+/datum/admins/proc/rouny_all()
+	set name = "Toggle Rouny All"
+	set category = "Fun"
+	set desc = "Turns all living xenos into rouny versions of themselves"
+
+	if(!check_rights(R_ADMIN))
+		return
```
Comment (TiviPlus) [inline code/modules/admin/fun_verbs.dm:77]: R_FUN please

File: code/modules/admin/fun_verbs.dm [hunk 2]
```diff
@@ -68,6 +68,19 @@
 	log_admin("[key_name(usr)] created a Queen Mother report: [input]")
 	message_admins("[ADMIN_TPMONTY(usr)] created a Queen Mother report.")
 
+/datum/admins/proc/rouny_all()
+	set name = "Toggle Rouny All"
+	set category = "Fun"
+	set desc = "Turns all living xenos into rouny versions of themselves"
```
Comment (TiviPlus) [inline code/modules/admin/fun_verbs.dm:74]: should say it toggles

**Review by TiviPlus — CHANGES_REQUESTED** [review]
rest looks ok I guess

**Review by TiviPlus — APPROVED** [review]
kuro

**Comment by SpiderPsycho** [comment]
If the Praetorian sprite is the one with the femboy socks I will be disappointed. There are no femboys in the Xenomorph hive, they are all female. Therefore, the Praetorian should have lacy stockings.

**Comment by AtlasRules** [comment]
> If the Praetorian sprite is the one with the femboy socks I will be disappointed. There are no femboys in the Xenomorph hive, they are all female. Therefore, the Praetorian should have lacy stockings.

jesse what the fuck are you talking about

**Pull request merged**